keyboards/gray_studio/space65/space65.h

```diff
@@ -17,8 +17,6 @@
 
 #include "quantum.h"
 
-#define XXX KC_NO
```

Comment:
Why don't you like `XXX`, again? I think it leads to better visual alignment.

Reply:
I remember noroadsleft in a PR of mine [saying](https://github.com/qmk/qmk_firmware/pull/4704#pullrequestreview-187576269) about `KC_NO` and `KC_TRNS` being already defined in the quantum code and therefore don't need to be defined again for the sake of uniformity

Reply:
Not the same thing since this is a board header rather than a keymap file. `XXX` is not a standard alias (yet).

Reply:
We should probably standardize this at some point. At the moment, around 35 boards define `XXX` in their headers. Some boards use `xxx` and `_x_` instead.

Reply:
I explicitly set it to KC_NO.

Reply:
Yeah, my comment was pertaining to keymaps rather than header files. I don't mind `XXXXXXX` or `_______` in keymaps; in fact, I use them myself.

For header files though, I've gotten to the point that I prefer using `KC_NO` directly, rather than `XXX`, `_x_`, or any of the other common variants. If we're going to standardize something for that usage, I'd be okay with standardizing on `XXX`, as I prefer that to any of the other variants I've seen, but I avoid it specifically because it's *not* standard.